composable_kernel/include/tensor_operation/reduction_functions_blockwise.hpp

```diff
@@ -23,247 +23,145 @@
  * SOFTWARE.
  *
  *******************************************************************************/
-#ifndef CK_REDUCTION_FUNCTIONS_BLOCKWISE_HPP
-#define CK_REDUCTION_FUNCTIONS_BLOCKWISE_HPP
+#ifndef CK_REDUCTION_FUNCTIONS_PARTITIONED_BLOCKWISE_HPP
```

Comment:
What is the difference between this class and following classes?

composable_kernel/include/tensor_operation/gridwise_2d_reduction_threadwise.hpp
composable_kernel/include/tensor_operation/gridwise_2d_reduction_multiblock_atomic_add.hpp
...